baremaps-core/src/main/java/org/apache/baremaps/iploc/dto/InetnumLocationDto.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.baremaps.iploc.dto;
+
+
+
+import org.apache.baremaps.iploc.data.InetnumLocation;
+import org.apache.baremaps.iploc.data.Ipv4;
+
+public class InetnumLocationDto {
+
+  private final String address;
+
+  private final String ipv4Start;
+
+  private final String ipv4End;
+
+  private final double latitude;
+
+  private final double longitude;
+
+  private final String network;
+
+  private final String country;
+
+  public InetnumLocationDto(InetnumLocation inetnumLocation) {
+
```

Comment:
can we remove these line returns in the constructor?

Reply:
I have removed them.